float64 pos_x
float64 pos_y
float64 pos_theta
uint8 weed_counter
string status
